core/shared/src/main/scala/zio/Chunk.scala

```diff
@@ -1100,6 +1112,12 @@ sealed abstract class Chunk[+A] extends ChunkLike[A] { self =>
 
 object Chunk extends ChunkFactory with ChunkPlatformSpecific {
 
+  sealed trait Endianness
```

Comment:
I am not sure, if this is a good place for it.

Reply:
idk about that. `BitChunk` is the byte version. And the only one that does not care about `Endianness`.

Reply:
See my comment below regarding potentially renaming `BitChunk` to `BitChunkByte`.